prov/efa/src/efa_data_path_direct_entry.h

```diff
@@ -428,6 +428,7 @@ static inline int efa_data_path_direct_start_poll(struct efa_ibv_cq *ibv_cq,
 
 static inline uint32_t efa_data_path_direct_wc_read_qp_num(struct efa_ibv_cq *ibv_cq)
 {
+	assert(ibv_cq->data_path_direct.cur_cqe != NULL);
```

Comment:
nit: why not just `assert(ibv_cq->data_path_direct.cur_cqe)`, similar to other asserts

Reply:
Changed. I don't feel strongly here.